config/routes.rb

```diff
@@ -23,6 +23,7 @@ def redirect(path, options = { prefix: Whitehall.router_prefix })
       resources :users, only: %i[index show edit update]
 
       get "republishing" => "republishing#index", as: :republishing_index
+      get "republishing/:document_slug/confirm" => "republishing#confirm", as: :confirm_republishing
```

Comment:
I'm curious if we need these `as`es here

Reply:
We've decided to keep these to make the routes a bit more readable.